go.mod

```diff
@@ -64,14 +64,14 @@ require (
 	go.uber.org/atomic v1.7.0 // indirect
 	go.uber.org/multierr v1.6.0 // indirect
 	go.uber.org/zap v1.24.0 // indirect
-	golang.org/x/net v0.7.0 // indirect
+	golang.org/x/net v0.31.0 // indirect
 	golang.org/x/oauth2 v0.0.0-20220223155221-ee480838109b // indirect
-	golang.org/x/sys v0.5.0 // indirect
-	golang.org/x/term v0.5.0 // indirect
-	golang.org/x/text v0.7.0 // indirect
+	golang.org/x/sys v0.27.0 // indirect
```

Comment:
these are indirect dependency required by package e.g. client-go, so it is not appropriate to change the indirect dependency. if the dependency version do bother you, you should upgrade the client-go package. the k8s and gateway-api dependency had been upgraded recently(https://github.com/openkruise/rollouts/pull/237/), yet the changes are not release yet. so further upgrade can be schedule for future, can you submit an issue, about the package requirement in your environment?

Reply:
1. fix the vulnerability of golang.org/x/net  CVE link: https://avd.aquasec.com/nvd/2022/cve-2022-27664/
2. fix the vulnerability of golang.org/x/text CVE link: https://avd.aquasec.com/nvd/2021/cve-2021-38561/